source/all.adoc

```diff
@@ -235,6 +235,7 @@ The final data store will be selected from the three options mentioned above (Re
 === Data Models
 
 A *dataset* is defined as a collection of data records and their associated information content (e.g., use, discovery, provenance metadata). In the E-SOH context, we consider the (Near-) Real-Time (NRT) data as extracts of externally available datasets like, e.g., climate timeseries. We refer to these datasets as "parent" datasets, whereas the extracts are referred to as "child" datasets.
```

Comment:
Actually, we need to reconsider this a bit as well. Since e-soh is NRT, the observations cannot be expected to exist in a climate dataset yet, and it will be difficult to connect them. I suggest to just remove the two last sentences here:
```suggestion
==== Dataset

A *dataset* is defined as a collection of data records and their associated information content (e.g., use, discovery, provenance metadata).
```